tools/build/Makefile.rules

```diff
@@ -405,3 +405,11 @@ $(KCONFIG_AUTOHEADER): $(DEPENDENCY_CACHE) $(KCONFIG_CONFIG) $(obj)/conf
 	$(Q)echo "     "GEN"   "$@
 	$(Q)mkdir -p $(KCONFIG_INCLUDE)config $(KCONFIG_INCLUDE)generated
 	$(Q)$(obj)/conf -s --silentoldconfig Kconfig
+
+include $(top_srcdir)bindings/python/bindings.mk
```

Comment:
it's better to avoid this different makefile, follow the pattern used by other soletta native parts, just define a bindings/python/Makefile and we'll create a rule to `.c -> .pyx` and it will be done automatically. We have similar rules to generate `.c -> .json` inside this file, we can base on that.